linkerd/protocol/h2/src/integration/scala/io/buoyant/linkerd/protocol/h2/ForwardClientCertTest.scala

```diff
@@ -1,19 +1,28 @@
 package io.buoyant.linkerd.protocol.h2
 
+import java.io.FileInputStream
+import java.security.MessageDigest
+import java.security.cert.CertificateFactory
+import javax.xml.bind.DatatypeConverter.printHexBinary
```

Comment:
Linkerd style is to group all imports together (no blank lines between imports) and sort in alphabetical order.